00_Intro/bedrock_boto3_setup.ipynb

```diff
@@ -432,6 +437,21 @@
     "With some theory out of the way, let's see the models in action! Run the cells below to see basic, synchronous example invocations for each model:"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6a0a79b9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bedrock_runtime = bedrock.get_bedrock_client(\n",
+    "    assumed_role=os.environ.get(\"BEDROCK_ASSUME_ROLE\", None),\n",
+    "    endpoint_url=os.environ.get(\"BEDROCK_ENDPOINT_URL\", None),\n",
```

Comment:
Since, Bedrock is GA, we no longer need to provide the bedrock endpoint url. Can you please verify and remove it. only assume role and region should work.

Reply:
Updated